ConfigSpace/hyperparameters.pyx

```diff
@@ -1581,6 +1941,45 @@ cdef class NormalIntegerHyperparameter(IntegerHyperparameter):
                 neighbors.append(new_value)
         return neighbors
         
+    def _compute_normalization(self):
+        if self.lower is None:
+            warnings.warn('Cannot normalize the pdf exactly for a NormalIntegerHyperparameter'
+            f'{self.name} without bounds. Skipping normalization for that hyperparameter.')
```

Comment:
You'll need a space here, `f' {self.name}'` as there's non on the preceeding line.